app/models/candidate_interface/satisfaction_survey_form.rb

```diff
@@ -0,0 +1,26 @@
+module CandidateInterface
+  class SatisfactionSurveyForm
+    include ActiveModel::Model
+
+    attr_accessor :question, :response
+
+    validates :question, presence: true
```

Comment:
Should we validate for the `inclusion` of the question in the set of answers we expect?

Reply:
That's in a follow up PR 👍

Reply:
![image](https://user-images.githubusercontent.com/42515961/78166879-68eab880-7445-11ea-8658-0e646bed38da.png)